src/components/Faq/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {FaqStyleTitle, FaqStyleContent} from './styles'
+
+class FAQ extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      clickedOn: false
+    }
+    this.handleClick = this.handleClick.bind(this);
+  }
+  
+  handleClick() {
+    // var content = this.nextElementSibling;
```

Comment:
I've heard its good practice to remove all of your commented out code before pushing up to production. It helps keep the code clean. So I would recommend going through and removing all of the unused code.